lib/ocamlorg_web/graphql.ml

```diff
@@ -0,0 +1,330 @@
+        ; field
+            "description"
+            ~doc:"The description of the package"
+            ~args:Arg.[]
+            ~typ:(non_null string)
+            ~resolve:(fun _ p ->
+              let info = Package.info p in
+              info.Package.Info.description)
+        ; field
+            "license"
+            ~doc:"The license of the package"
+            ~args:Arg.[]
+            ~typ:(non_null string)
+            ~resolve:(fun _ p ->
+              let info = Package.info p in
+              info.Package.Info.license)
+        ; field
+            "homepage"
+            ~doc:"The homepage of the package"
+            ~args:Arg.[]
+            ~typ:(non_null (list (non_null string)))
+            ~resolve:(fun _ p ->
+              let info = Package.info p in
+              info.Package.Info.homepage)
+        ; field
+            "tags"
+            ~doc:"The tags of the package"
+            ~args:Arg.[]
+            ~typ:(non_null (list (non_null string)))
+            ~resolve:(fun _ p ->
+              let info = Package.info p in
+              info.Package.Info.tags)
+        ; field
+            "authors"
+            ~doc:"The authors of the package"
+            ~args:Arg.[]
+            ~typ:(non_null (list (non_null owners)))
+            ~resolve:(fun _ p ->
+              let info = Package.info p in
+              get_owners info.Package.Info.authors)
+        ; field
+            "maintainers"
+            ~doc:"The maintainers of the package"
+            ~args:Arg.[]
+            ~typ:(non_null (list (non_null owners)))
+            ~resolve:(fun _ p ->
+              let info = Package.info p in
+              get_owners info.Package.Info.maintainers)
+        ; field
+            "dependencies"
+            ~doc:"The dependencies of the package"
+            ~args:Arg.[]
+            ~typ:(non_null (list (non_null deps)))
+            ~resolve:(fun _ p ->
+              let info = Package.info p in
+              get_deps info.Package.Info.dependencies)
+        ; field
+            "depopts"
+            ~doc:"The depopts of the package"
+            ~args:Arg.[]
+            ~typ:(non_null (list (non_null deps)))
+            ~resolve:(fun _ p ->
+              let info = Package.info p in
+              get_deps info.Package.Info.depopts)
+        ; field
+            "conflicts"
+            ~doc:"The conflicts of the package"
+            ~args:Arg.[]
+            ~typ:(non_null (list (non_null deps)))
+            ~resolve:(fun _ p ->
+              let info = Package.info p in
+              get_deps info.Package.Info.conflicts)
+        ; field
+            "url"
+            ~doc:"The url of the package"
+            ~args:Arg.[]
+            ~typ:url
+            ~resolve:(fun _ p ->
+              let info = Package.info p in
+              info.Package.Info.url)
+        ]))
+
+let response =
+  Graphql_lwt.Schema.(
+    obj "response" ~fields:(fun _ ->
```

Comment:
I would specialize the name of this field to the package response. E.g. `packages_result`